projects/Mallard/src/navigation/navigators/underlay/transition.tsx

```diff
@@ -12,8 +12,11 @@ const issueScreenToIssueList = (sceneProps: NavigationTransitionProps) => {
     const sceneIndex = scene.index
     const { height: windowHeight, width } = Dimensions.get('window')
     const isTablet = width >= Breakpoints.tabletVertical
+    const isPhone = width >= Breakpoints.phone
 
-    const finalTranslate = windowHeight - 80
+    const finalTranslate = isPhone
+        ? windowHeight - windowHeight / 4
+        : windowHeight - windowHeight / 5.6
```

Comment:
I am concerned about these magic numbers. Can we start to centralise some of this and keep these in line with the font sizes for example?

Reply:
What would you suggest?

Reply:
I'm not sure in this case why we would align these to font sizes.